### DISCUSSION TITLE
poppies popping up everywhere...

### DISCUSSION THREAD

**Discussion by fvbeers** [tannerhelland/PhotoDemon#407] [Show and tell]
Yes there are right now here in the Netherlands.
And I love them....
So I took a macro photo I once took and went playing with it in version 9 pre-alpha build 1580.
You can see 6 results in my [Demon album](https://flic.kr/s/aHsk72vDJX) on Flickr:

cheers,
Frans

**Comment by tannerhelland** [comment]
Thank you for sharing, @fvbeers !  I love to see beautiful poppies  🙏
